keras_nlp/tests/doc_tests/docstring_test.py

```diff
@@ -114,6 +114,9 @@ def test_fenced_docstrings():
             "keras_nlp.models.xlm_roberta.xlm_roberta_preprocessor",
             "keras_nlp.models.f_net.f_net_preprocessor",
             "keras_nlp.models.f_net.f_net_tokenizer",
+            "keras_nlp.tokenizers.byte_pair_tokenizer",
```

Comment:
I think these three additions should be in the "base classes" block a few lines up.

Reply:
Made the required changes @mattdangerw. Please have a look